### PR TITLE
Add support for postgres on io.js

### PR DESCRIPTION
**Pull request by ruimarinho**
Adds support for postgres on io.js.

* Upgrade pg@4.2.0
* Upgrade pg-native@1.8.0

Two important notes:

* There is a new `SET` that increases the message level to `warning` (up from `notice`) since the upgraded pg client is now capable of bubbling these messages.
* There is something going on with timers that I had to workaround with promises. The tests where sinon's fake clock are used hang for some reason. This doesn't seem related to io.js since the lib tests used by sinon to fake timers are passing on the latest io.js version. If anyone is able to figure that out, I'd appreciate it.

### CODE REVIEW

File: test/integration/model.test.js [hunk 1]
```diff
@@ -1010,9 +1008,11 @@ describe(Support.getTestDialectTeaser('Model'), function() {
         expect(this.updatedAt).to.equalTime(users[2].updatedAt); // All users should have the same updatedAt
 
         // Pass the time so we can actually see a change
-        this.clock.tick(1000);
-
-        return this.User.update({username: 'Bill'}, {where: {secretValue: '42'}});
+        return new this.sequelize.Promise(function(resolve) {
```
Comment (janmeier) [inline test/integration/model.test.js]: This could use `return this.sequelize.Promise.timeout(1000)`

**Comment by janmeier** [comment]
I've experienced problems with the combination of promises and fake timers as well. If you change it to `Promise.timeout` it should be fine

**Comment by ruimarinho** [comment]
@janmeier fixed and rebased.

**Comment by janmeier** [comment]
@ruimarinho merged and thanks sent :)

**Pull request merged**